docs/public_api/api_reference.md

```diff
@@ -0,0 +1,117 @@
+Public API - Reference
```

Comment:
These are all generated manually.
For the future, we should consider auto-generating these through a script.